src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xencoder/dropdownList/AlgorithmDropdown.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *  * Copyright (c) 2022 AVI-SPL, Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.avdevices.encoderdecoder.haivision.xencoder.dropdownList;
```

Comment:
The packagename should be com.avispl.symphony.dal.avdevices.encoderdecoder.haivision.xencoder.dropdownlist